x-pack/plugins/alerting/server/rules_client/methods/update.ts

```diff
@@ -88,6 +89,18 @@ async function updateWithOCC<Params extends RuleTypeParams>(
     alertSavedObject = await context.unsecuredSavedObjectsClient.get<RawRule>('alert', id);
   }
 
+  try {
+    if (alertSavedObject.attributes.enabled) {
+      await validateScheduleLimit({
```

Comment:
only do this validation if you see that the interval changed

Reply:
https://github.com/elastic/kibana/pull/164791/commits/74a53604d4cad622dc43ad41d53d08f8cddcb859